bin/vscode-setup

```diff
@@ -0,0 +1,31 @@
+#! /usr/bin/env bash
+
+# DOC: Open an sbt CLI in the dev environment container.
+
+source bin/lib.sh
+
+bin/pull-image
+
+bin/sbt makePom && cp universal-application-tool-0.0.1/target/scala-*/*.pom universal-application-tool-0.0.1/pom.xml
```

Comment:
suggestion (blocking): use multiple lines for styleguide conformance to line length. also with `set -e` (done in bin/lib.sh) no need to guard the `cp` command since if `bin/sbt makePom` fails the script will exit.
```suggestion
bin/sbt makePom
cp universal-application-tool-0.0.1/target/scala-*/*.pom \
  universal-application-tool-0.0.1/pom.xml
```